nilearn/interfaces/fmriprep/load_confounds_utils.py

```diff
@@ -327,32 +365,44 @@ def load_confounds_file_as_dataframe(confounds_raw_path):
         confounds_raw_path, delimiter="\t", encoding="utf-8"
     )
 
-    # check if the version of fMRIprep (>=1.2.0) is supported based on
-    # header format. 1.0.x and 1.1.x series uses camel case
-    if any(is_camel_case(col_name) for col_name in confounds_raw.columns):
-        raise ValueError(
-            "The confound file contains header in camel case. "
-            "This is likely the output from 1.0.x and 1.1.x series. "
-            "We only support fmriprep outputs >= 1.2.0."
-            f"{confounds_raw.columns}"
-        )
+    if not flag_tedana:
```

Comment:
can you reverse the logic:
- deal with tedana first
- return early and do not use an else block